lib/Elastica/Aggregation/AbstractSimpleAggregation.php

```diff
@@ -32,6 +32,11 @@ public function setScript($script)
      */
     public function toArray()
     {
+        if (!$this->hasParam('field') && !$this->hasParam('script')) {
+            throw new \LogicException(
```

Comment:
In Elastica there is no usage of LogicException, I think better to change it to \Elastica\Exception\InvalidException to be on one wave with similar checks.